src/System.Numerics.Vectors/tests/GenericVectorTests.tt

```diff
@@ -1869,4 +1891,4 @@ namespace System.Numerics.Tests
         }
         #endregion
     }
-}
\ No newline at end of file
+}
```

Comment:
Tried a lot to avoid this change. But don't know how to stop this automatic edit in VS2017 Community edition.

Reply:
I typically ignore these changes if VS does them automatically.  I wouldn't worry about it.